compiler/dyno/test/uast/testStringify.cpp

```diff
@@ -45,6 +46,31 @@ using namespace parsing;
       assert(!ss.str().empty());                          \
     }
 
+// Helper macro for testing output from printUserString
+#define TEST_USER_STRING(funcDef, val)                               \
+    {                                                                \
+      std::ostringstream ss;                                         \
+      auto parseResult = parser->parseString("test3.chpl", funcDef); \
+      auto mod = parseResult.singleModule();                         \
+      auto funcDecl = mod->stmt(0)->toFunction();                    \
+      assert(funcDecl);                                              \
+      printUserString(ss, funcDecl);                                 \
+      std::cerr << ss.str() << std::endl;                            \
+      assert(ss.str() == val);                                       \
+    }
+
+
+#define TEST_CHPL_SYNTAX(src, val) \
+{\
```

Comment:
Nit: braces, either align them as is done above or don't align them at all, and just put them at the end of the code line. Personally I prefer the second, because you have to do less work in the event that you need to edit the macro.